web/vtadmin/src/components/routes/tablet/Advanced.tsx

```diff
@@ -0,0 +1,159 @@
+import React, { useState } from 'react'
+import { useHistory, useParams } from 'react-router-dom';
+import { deleteTablet, reparentTablet, startReplication, stopReplication } from '../../../api/http';
+import { vtadmin } from '../../../proto/vtadmin';
+import { isPrimary } from '../../../util/tablets';
+import { Icon, Icons } from '../../Icon';
+import { success, warn } from '../../Snackbar';
+import { TextInput } from '../../TextInput';
+
+interface AdvancedProps {
+    tablet: vtadmin.Tablet | undefined
+}
+
+interface RouteParams {
+    alias: string;
+    clusterID: string;
+}
+
+const Advanced: React.FC<AdvancedProps> = ({ tablet }) => {
+    const { clusterID, alias } = useParams<RouteParams>();
+    const history = useHistory()
+
+    // DeleteTablet
```

Comment:
Each API is defined within the Advanced.tsx file; while it is long/repetitive I think it's clearer to keep them here instead of breaking them out into components or DRYing it up.